src/endpoint.cpp

```diff
@@ -706,6 +714,10 @@ bool UartEndpoint::setup(UartEndpointConfig conf)
         }
     }
 
+    if (conf.ignorecompid) {
```

Comment:
This if-statement isn't needed as you have a default value in the config struct, so the value always exists. (Applies to all three occurrences.)